### PR TITLE
revamp piesno docstring

### PR DESCRIPTION
**Pull request by samuelstjean**
I was looking at finding why it does throw some divide by zero warning, even though they do not seem to affect the outcome. Seems like maybe when a slice has a lot of high values, the median is not zero but there might be no noise also, so I need to further check it out if it is just safe to squelch it instead.

For now some rewording and I need to also re-read the example to check it still makes sense, but I had this one for a few weeks already, so there is.

### CODE REVIEW

File: dipy/denoise/noise_estimate.py [hunk 1]
```diff
@@ -149,10 +146,11 @@ def _piesno_3D(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5,
     -----------
     data : ndarray
         The magnitude signals to analyse. The last dimension must contain the
-        same realisation of the volume, such as dMRI or fMRI data.
+        same realisation of the volume.
```
Comment (arokem) [inline dipy/denoise/noise_estimate.py]: Maybe use the same wording as above? ("e.g. volumes collected during dMRI or fMRI acquisitions.")

**Comment by samuelstjean** [comment]
I still want to figure out why it does stuff like that before though
~~~bash
/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/dipy/denoise/noise_estimate.py:248: RuntimeWarning: divide by zero encountered in true_divide
  s = sum_m2 / (2 * K * sigma**2)
/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/dipy/denoise/noise_estimate.py:248: RuntimeWarning: invalid value encountered in true_divide
  s = sum_m2 / (2 * K * sigma**2)
/home/travis/miniconda/envs/test-environment/lib/python3.5/site-packages/dipy/denoise/noise_estimate.py:249: RuntimeWarning: invalid value encountered in less_equal
  mask[...] = np.logical_and(lambda_minus <= s, s <= lambda_plus)
~~~
probably nothing as it still works, but at least somehow squelching that would be useful.

**Comment by arokem** [comment]
The latter two might be because of nans or infs in the array?

**Comment by samuelstjean** [comment]
That the symptom, I'm trying ot figure out why the std value goes to zero sometimes in the first place

**Comment by Garyfallidis** [comment]
I am sorry but this or any other PRs from Sam will not be merged as decided and expressed clearly last year. Sam there is no reason for you making any more PRs in dipy. We discussed about this last year let's not get back to the same discussions. I am closing this one and will remove PIESNO from the documentation. You can copy PIESNO to your own github project and work on improvements there.

**Comment by samuelstjean** [comment]
not sure why you would want to remove it from the doc, it works but it
could be better worded.

2017-06-19 16:52 GMT+02:00 Eleftherios Garyfallidis <
notifications@github.com>:

**Pull request closed without merging**